mmv1/third_party/terraform/resources/resource_compute_instance_from_machine_image.go.erb

```diff
@@ -69,29 +69,9 @@ func computeInstanceFromMachineImageSchema() map[string]*schema.Schema {
 		Description: `Name or self link of a machine image to create the instance from on.`,
 	}
 
-	// Modifying the schema to disable disk overrides, due to an API bug (b/170964971)
```

Comment:
@edwardmedia Your fix seems unrelated to removing this code. While we probably should do so, have you tested that we are now able to override the boot_disk successfully?

Reply:
related code added back